bpf/lib/nodeport.h

```diff
@@ -510,6 +508,9 @@ static __always_inline int nodeport_lb6(struct __ctx_buff *ctx,
 	case CT_NEW:
 		ct_state_new.src_sec_id = SECLABEL;
 		ct_state_new.node_port = 1;
+#ifdef NATIVE_DEV_IFINDEX
```

Comment:
All these #ifdef's are required to compile bpf_lxc.c, as it depends on nodeport.h. Alternatively, I could define a dummy value in lxc_config.h.